### PR TITLE
rename options for linear regression

### PR DESCRIPTION
**Pull request by fqixiang**
fixed https://github.com/jasp-stats/INTERNAL-jasp/issues/1937#event-7066135770

### CODE REVIEW

File: inst/qml/RegressionLinear.qml [hunk 1]
```diff
@@ -100,46 +100,46 @@ Form
...
 			}
 
 			Group
 			{
 				CheckBox { name: "modelFit";					label: qsTr("Model fit");  checked: true		}
 				CheckBox { name: "rSquaredChange";				label: qsTr("R squared change")				}
-				CheckBox { name: "descriptives";				label: qsTr("Descriptives")					}
-				CheckBox { name: "partAndPartialCorrelations";	label: qsTr("Part and partial correlations")	}
-				CheckBox { name: "collinearityDiagnostics";		label: qsTr("Collinearity diagnostics")		}
+				CheckBox { name: "descriptive";				label: qsTr("Descriptives")					}
```
Comment (Kucharssim) [inline inst/qml/RegressionLinear.qml:115]: I think here you could make an exception and just go with `descriptives`

**Review by Kucharssim — CHANGES_REQUESTED** [review]
looks very good, just minor remarks.

in `steppingMethodCriteriaType`:

- `usePValue` -> `pValue`
- `useFValue` -> `fValue`

`missingValues` -> `naAction`:
- `caseListsieExclusion` -> `listwise`
- `casesPairwiseExclusion` -> `pairwise`

File: inst/Upgrades.qml [hunk 2]
```diff
@@ -175,80 +175,150 @@ Upgrades
...
+		ChangeRename { from: "wlsWeights";					                                to: "weights"							                    }
+		ChangeRename { from: "includeConstant";					                            to: "interceptTerm"							                }
+		ChangeRename { from: "regressionCoefficientsEstimates";					            to: "coefficientEstimate"							        }
+		ChangeRename { from: "regressionCoefficientsBootstrapping";				            to: "coefficientBootstrap"			                        }
+		ChangeRename { from: "regressionCoefficientsBootstrappingReplicates";				to: "coefficientBootstrapReplicates"			            }
+		ChangeRename { from: "regressionCoefficientsConfidenceIntervals";					to: "coefficientCi"							                }
+		ChangeRename { from: "regressionCoefficientsConfidenceIntervalsInterval";			to: "coefficientCiLevel"							        }
+		ChangeRename { from: "regressionCoefficientsCovarianceMatrix";					    to: "covarianceMatrix"							            }
+		ChangeRename { from: "VovkSellkeMPR";					                            to: "vovkSellke"							                }
+		ChangeRename { from: "descriptives";					                            to: "descriptive"							                }
```
Comment (Kucharssim) [inline inst/Upgrades.qml:267]: Delete this rename since we don't rename this option anymore

File: inst/Upgrades.qml [hunk 3]
```diff
@@ -175,80 +175,150 @@ Upgrades
...
+				}
+			}
+		}
+
+		ChangeJS
+		{
+			name:		"steppingMethodCriteriaType"
+			jsFunction:	function(options)
+			{
+				switch(options["naAction"])
```
Comment (Kucharssim) [inline inst/Upgrades.qml:313]:
```suggestion
				switch(options["steppingMethodCriteriaType"])
```

File: inst/Upgrades.qml [hunk 4]
```diff
@@ -175,80 +175,150 @@ Upgrades
...
+				}
+			}
+		}
+	}
+
+	Upgrade
+	{
+		functionName:		"RegressionLinear"
+		fromVersion:		"0.16.3"
+		toVersion:			"0.16.4"
```
Comment (Kucharssim) [inline inst/Upgrades.qml:256]:
```suggestion
		fromVersion:		"0.16.4"
		toVersion:			"0.17.0"
```

**Review by Kucharssim — CHANGES_REQUESTED** [review]
looks great, almost there! Just last fixes to the upgrader

File: inst/Upgrades.qml (hunk 4 above)
Comment (Kucharssim) [reply inst/Upgrades.qml:256]: Can you fix this? Just need to make sure that the upgrader uses the correct versions

Comment (fqixiang) [reply inst/Upgrades.qml:256]: Yes, I'm just waiting for the logistic regression PR to be merged first before I do this PR. There seems to be some conflicts.

Comment (Kucharssim) [reply inst/Upgrades.qml:256]: ok, makes sense :) I just saw you marked this as resolved, but we still need to do that. 

Anyway, the logistic regression is merged now so you can fix the conflicts

Comment (fqixiang) [reply inst/Upgrades.qml]: done :)

**Review by Kucharssim — APPROVED** [review]

**Pull request merged**